kolibri/plugins/coach/assets/src/views/create-exam-page/index.vue

```diff
@@ -108,6 +110,7 @@
 
   const ExamActions = require('../../state/actions/exam');
   const ExamModals = require('../../examConstants').Modals;
+  const shuffle = require('lodash.shuffle');
```

Comment:
I've seen recent moves to 'cherrypick' lodash functions from the full library via `lodash/shuffle`. That way, we still don't bundle all of lodash, but we also don't need to continually add entries to package.json whenever there's a new utility we want to use.